devicetypes/smartthings/zwave-temp-humidity-sensor.src/zwave-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,213 @@
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "temperature", type: "generic", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.temperature", key: "PRIMARY_CONTROL") {
+				attributeState "temperature", label: '${currentValue}°',
+					backgroundColors: [
+						[value: 31, color: "#153591"],
+						[value: 44, color: "#1e9cbb"],
+						[value: 59, color: "#90d2a7"],
+						[value: 74, color: "#44b621"],
+						[value: 84, color: "#f1d801"],
+						[value: 95, color: "#d04e00"],
+						[value: 96, color: "#bc2323"]
+					]
+			}
+		}
+		valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+			state "humidity", label: '${currentValue}% humidity', unit: ""
+		}
+		valueTile("dewPoint", "device.dewPoint", inactiveLabel: false, width: 2, height: 2) {
+			state "dewPoint", label: '${currentValue}° dewPoint', unit: ""
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "temperature", "humidity", "dewPoint"
+		details(["temperature", "humidity", "dewPoint", "battery"])
+	}
+}
+
+def updated() {
+    configure()
+}
+
+def installed() {
+    sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def configure() {
+	response([
+		secure(zwave.batteryV1.batteryGet()),
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05)), // humidity
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01)), // temperature
+		"delay 500",
+		secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x0B)), // dew point
+		"delay 500",
+		secure(zwave.wakeUpV2.wakeUpNoMoreInformation())
+	])
+}
+
+def parse(String description) {
+	def results = []
+
+	if (description.startsWith("Err")) {
+		results += createEvent(descriptionText: description, displayed: true)
+	} else {
+		def cmd = zwave.parse(description)
+		if (cmd) {
+			results += zwaveEvent(cmd)
+		}
+	}
+
+	log.debug "parse() result ${results.inspect()}"
+
+	return results
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensorbinaryv1.SensorBinaryReport cmd) {
+	// ignore, to prevent override of SensorBinaryReport
+	[]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+	log.debug "Event: ${cmd.event}, Notification type: ${cmd.notificationType}"
+	def result = []
+	def value
+	def description
+
+	if (cmd.notificationType == 0x10) {  // Mold Environment Detection
```

Comment:
Is it presented to user in any way (excluding IDE)?

Reply:
not yet, it is discussed here [WWST-6928](https://smartthings.atlassian.net/browse/WWST-6928)